### PR TITLE
[MediaPlayer] Add Xcode 11 Beta 1-3 bindings

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

**Comment by tj-devel709** [comment]
Dependent on AVFoundation. Has AVMediaSelectionGroup and AVMediaSelectionOption set to NSObject for the watch currently. https://github.com/xamarin/xamarin-macios/issues/6597

File: src/mediaplayer.cs [hunk 1]
```diff
@@ -1280,12 +1280,17 @@ interface MPVolumeView {
 		[Export ("initWithFrame:")]
 		IntPtr Constructor (CGRect frame);
 
+		[Deprecated (PlatformName.iOS, 13, 0, message: "Use AVRoutePickerView instead")]
```
Comment (joconte) [inline src/mediaplayer.cs]: Maybe add a ```.```at the end of the message for consistency.

Comment (tj-devel709) [reply src/mediaplayer.cs]: Good catch! Thanks :)

File: src/mediaplayer.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@
 using UIKit;
 #endif
 using System;
-#if WATCH
+#if WATCH // TODO Will be removed after CoreMedia and AVFoundation are updated
 using AVMediaSelectionGroup = Foundation.NSObject;
 using AVMediaSelectionOption = Foundation.NSObject;
 #endif
```
Comment (spouliot) [inline src/mediaplayer.cs]: ^ that creates unusable API and it's too easy to forget
Better not update `watchOS-MediaPlayer.todo` so it stays visible (and under our radar).

IOW just do the iOS parts until other parts are ready

Comment (tj-devel709) [reply src/mediaplayer.cs]: Okay I will do that!

**Pull request merged**